main_test.go

```diff
@@ -15,10 +16,9 @@ func TestMainHandlerWhenOk(t *testing.T) {
 	responseRecorder := httptest.NewRecorder()
 	handler := http.HandlerFunc(mainHandle)
 	handler.ServeHTTP(responseRecorder, req)
+	status := responseRecorder.Code
 
-	if status := responseRecorder.Code; status != http.StatusOK {
-		t.Fatalf("expected status code: %d, got %d", http.StatusOK, status)
-	}
+	require.Equalf(t, status, http.StatusOK, "expected status code: %d, got %d", http.StatusOK, status)
```

Comment:
На втором месте нужно ставить что хотим увидеть, а на третьем, что проверяем.
На самом тест это не влияет, но когда он упадет, то вывод будет неправильный

Простые комментарии можно не оставлять, у тестов красивый вывод, можешь сам посмотреть если поставить не тот код.
Так же можно отдельную переменную не создавать под это.

В принципе такого вполне достаточно
```suggestion
	require.Equalf(t, http.StatusOK, responseRecorder.Code)
```